partcad-cli/src/partcad_cli/click/commands/reset.py

```
@@ -1,23 +1,27 @@
 def cli(ctx: Context) -> None:
-    with Process("Reset", "this"):
-        for _, package_info in ctx.config_obj.get("dependencies", {}).items():
-            import_type = package_info["type"]
+    with Process("Reset", ctx.name):
+        cached_import_types = ["git", "tar"]
+        for import_type in cached_import_types:
             cache_dir = os.path.join(user_config.internal_state_dir, import_type)
             if os.path.exists(cache_dir):
-                info(f"Removing cached {import_type} dependencies: '{cache_dir}'")
-                shutil.rmtree(cache_dir)
+                with Action(f"Repos", import_type):
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)